server/app.js

```diff
@@ -24,7 +26,7 @@ const version = moment.now().toString()
 const production = process.env.NODE_ENV === 'production'
 const testMode = process.env.NODE_ENV === 'test'
 
-module.exports = function createApp({ signInService, formService }) {
+module.exports = function createApp({ signInService, formService, nomisService }) {
```

Comment:
not a fan of this name - which nomis service does it correspond too?  We've renamed nomis to dps too, perhaps should be `elite2Service`.  Not precious though, would just like to be consistent

Reply:
fixed this: 
 * nomisClientBuilder -> elite2ClientBuilder
 * nomisService -> offenderService